content/integrations/faq/list-of-api-source-attribute-value.md

```diff
@@ -0,0 +1,206 @@
+---
```

Comment:
This feels like it should be generated from dogweb AND postgres.  
Keep in mind new sourcetypes are primarily added in the database.

Reply:
@irabinovitch agreed, I have added a card in the trello board. 
But can we merge this in the meantime?